mystery_o_matic/output/latex/__init__.py

```
@@ -0,0 +1,148 @@
+
+    for room, name in locations.indices.items():
+        names_txt[room] = locations.names['en'][name]
+
+    final_locations_map = {}
+    for c, p in mystery.final_locations.items():
+        c = create_template(c).substitute(names_txt)
+        p = create_template(p).substitute(names_txt)
+        final_locations_map[c] = p
+
+    representations_map = {}
+    for l, r in locations.representations.items():
```

Ruff

Ambiguous variable name: `l`

(E741)